src/main/java/com/epam/tc/hw9/dto/YandexSpellerDto.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw9.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+public class YandexSpellerDto {
```

Comment:
by param 'String [] s' I could make a conclusion that this DTO related to schema of response. Could it be represented in name of class?